app.py

```diff
@@ -40,12 +40,17 @@ def training():  # call method training
 def query():  # call method training
 
   rq = request.json
-  default_nn = config.default_nn()
   result = {}
+
+  if "n" in rq:
```

Comment:
let's keep it descriptive so that any one can understand what the parameters means